src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -47,6 +47,13 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
           _pd(pd),
           _prim(pd) { }
 
+    typed_primitive_onednn_impl(const typed_program_node<PType>& arg,
+                                const PrimDescType& pd)
```

Comment:
Do we need to accept `pd`? It is not used... And please add a comment to show that it is for "optimized" kernel.

Reply:
Done. Added 'assert(arg.can_be_optimized());' in ctor to ensure it is optimized.